dbms/src/Storages/DeltaMerge/Segment.cpp

```diff
@@ -560,82 +559,24 @@ Segment::SegmentMetaInfos Segment::readAllSegmentsMetaInfoInRange( //
         LOG_INFO(
             log,
             "Finish building segments, target_range={} infos_size={} total_processed_segments={} "
-            "total_skipped_segments={} reused_agg={} use_cache={}",
+            "total_skipped_segments={} reused_agg={}",
             target_range.toDebugString(),
             segment_infos.size(),
             total_processed_segments,
             total_skipped_segments,
-            reused_agg.toString(),
-            use_cache);
+            reused_agg.toString());
         return std::make_pair(end_key_and_segment_ids, segment_infos);
     };
 
-    if (use_cache)
```

Comment:
what's `fap_use_segment_to_end_map_cache`, and why remove it?

Reply:
The settings is added in the cse branch to verified the performance whether this cache help performance. We've verified that `fap_use_segment_to_end_map_cache == true` is useless on the serverless env. And we expected we will not set it to true in the next-gen TiFlash. So remove the dead code of `fap_use_segment_to_end_map_cache == true`, and also not introducing this setting to the next gen tiflash.